client/web/reporter/src/mixins/report.js

```diff
@@ -95,14 +97,12 @@ export default {
     handleClone (report) {
       this.processing = true
       this.processingClone = true
-      const { handle, meta, sources, blocks, scenarios, labels } = report
+      const clonedReport = this.report.clone()
```

Comment:
There is no need to clone the report (this causes the list clone to fail)

Just prepare the variables which you get from the report and pass them to the reportCreate